src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_MULTIPLIER = 2;
+    private Node<K, V>[] nodesArray;
+    private int capacity;
+    private int size;
+
+    public MyHashMap() {
+        nodesArray = new Node[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+    }
+
+    private int getHash(Object key) {
+        return (key == null) ? 0 : Math.abs(key.hashCode() % capacity);
+    }
+
+    private void resize() {
+        size = 0;
+        Node<K, V>[] tempNodesArray = nodesArray;
+        capacity *= CAPACITY_MULTIPLIER;
+        nodesArray = new Node[capacity];
+        for (Node<K, V> node: tempNodesArray) {
+            while (node != null) {
+                put(node.key, node.value);
+                node = node.next;
+            }
+        }
+    }
 
     @Override
```

Comment:
it is better to use greater than or equals to sign in the condition
 if (size >= capacity * LOAD_FACTOR) {
        resize();
    }